server/package.json

```diff
@@ -24,6 +23,7 @@
     "nohm": "^2.2.3",
     "redis": "^3.0.2",
     "secp256k1": "^4.0.0",
+    "to-arraybuffer": "^1.0.1",
```

Comment:
Don't think we need to add this dependency.

Reply:
> The `Reader` class is designed to handle both hex string and array buffer during instantiation. This change adds an unnecessary dependency. Can you talk about the motivation for commit [657981c](https://github.com/WilfredTA/token_mint/commit/657981c65f82827e73cf2a9ac696dfb4a57b5c3d)?

The motivation is performance. String conversions take 400% more CPU time compared to buffer-to-buffer copies.

Reply:
In that case, the file can be read as a buffer w/o conversion to hex & therefore w/o needed to add another package

Reply:
The Node.js readFileSync() function reads the data into an instance of Buffer that is not directly compatible with ArrayBuffer. Buffer uses a shared underlying ArrayBuffer which can contain other data. This is some kind of optimization they use to boost performance internally. If you try to use it directly without proper conversion, you may get corrupted data. No error messages, just wrong data. I found that out the hard way. This module handles the conversion properly in a way that is portable between platforms.